.bazelrc

```diff
@@ -70,3 +70,15 @@ build:asan-jazzer --@rules_fuzzing//fuzzing:cc_engine_instrumentation=jazzer
 build:asan-jazzer --@rules_fuzzing//fuzzing:cc_engine_sanitizer=asan
 # Workaround for https://github.com/bazelbuild/bazel/issues/11128
 build:asan-jazzer --//fuzzing:cc_engine_sanitizer=asan
+
+# Coverage with Replay (C/C++ only)
+coverage --//fuzzing:cc_engine=//fuzzing/engines:replay
+coverage --@rules_fuzzing//fuzzing:cc_engine_instrumentation=none
+coverage --@rules_fuzzing//fuzzing:cc_engine_sanitizer=none
+coverage --instrument_test_targets
+coverage --action_env=BAZEL_USE_LLVM_NATIVE_COVERAGE=1
+coverage --action_env=GCOV=llvm-profdata-10
+coverage --action_env=BAZEL_LLVM_COV=llvm-cov-10
+coverage --combined_report=lcov
+coverage --experimental_use_llvm_covmap
+coverage --experimental_generate_llvm_lcov
```

Comment:
I think this is great for the rules_fuzzing project itself, but what does it mean for the other projects using the fuzzing rules? Will they have to copy-paste and maintain this snippet in their own `.bazelrc` files? Could we integrate these into the existing instrumentation framework somehow, or this is not worth the effort?

Reply:
I anticipate that projects would maintain their own configurations for using `bazel coverage`. 

These flags are not specific to rules_fuzzing, but are instead for gathering coverage on clang-compiled code in general, whether from a `cc_test` or a `cc_fuzz_test`.

Reply:
> I anticipate that projects would maintain their own configurations for using `bazel coverage`.

This might be suboptimal in the long-term. My concern is that if we need to make changes to this set of flags over time (e.g., as the Bazel coverage support evolves), it will be very difficult to propagate them to client libraries if they need to duplicate them in their `.bazelrc` files.

Ideally, we'd be somewhere where we could ask our users to simply define:

```
coverage --@rules_fuzzing//fuzzing:cc_engine=@rules_fuzzing//fuzzing/engines:replay
coverage --@rules_fuzzing//fuzzing:cc_engine_instrumentation=coverage
coverage --@rules_fuzzing//fuzzing:cc_engine_sanitizer=none
```

and things would just work out of the box.

In any case, this is probably good enough for *our own* coverage support, so we can move forward with it.